native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/rabbitmq-native/resource-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "resources":{
+  "includes":[
+    {
+      "pattern":"\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E"
```

Comment:
I think this resource class is not needed. Have you tried to remove this?

Reply:
Yes, you're correct. Fixed in 11ee1a34076e716314cbe0e9f49d6ff6f958e2d4. Verified.